docs/users_guide/ocean/test_groups/spherical_harmonic_transform.rst

```diff
@@ -0,0 +1,107 @@
+.. _ocean_spherical_harmonic_transform:
+
+spherical_harmonic_transform
+============================
+
+The ``spherical_harmonic_transform`` test group implements test cases
+for the spherical harmonic transformation implementations in 
+MPAS-Ocean. These tests involve computing the forward and inverse spherical 
+harmonic transform of a certain order for an analytical function.
+The truncation errors for each order are compared with the analytical 
+function.
+
+For ocean simulations, the spherical harmonic transformation capability
+is used to compute the self attraction and loading forcing for tides.
+
+Two different implementations are used in this test group:
+
+- The first is a "serial" algorithm which involves an interpolation
+  from the MPAS mesh to a Guassian grid and  then uses a fast shperical
+  harmonic library, SHTns, to compute the forward and inverse 
+  transformations. The approximate function is then interpolated back 
+  to the MPAS mesh.
+  Since the SHTns library dosen't use distributed memory parallelism,
+  the function values are first gathered onto a single MPI rank that
+  is responsible for the interpolation and forward and inverse 
+  transformations. Finally, a scatter is performed to return the local
+  approximate function values to each core.
+- The second implimentation is a "parallel" algoritm which does not use
+  a fast library, but instead computes the spherical harmonic transform
+  directly on the unstructured mesh using an MPI All_Reduce.
+
+In order for this test case to run properly for the "serial" algorithm,
+MPAS-Ocean must be compiled with the ``USE_SHTNS=true`` option. 
```

Comment:
Could you add that they should also have `USE_OPENMP=false`?  This just tripped me up when I tried building.

Reply:
Yes, thanks for catching that!